src/main/java/org/blitmatthew/Calculator.java

```diff
@@ -0,0 +1,4 @@
+package org.blitmatthew;
+
+public class Calculator {
```

Comment:
The Calculator is missing the methods to be used. Can we get the methods in before we merge the branch

Reply:
Yeah, give me about an hour and I'll have the methods completed